ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -328,13 +328,13 @@ def yield_data_models(
         if self.source_config.dbtConfigSource and dbt_objects.dbt_manifest:
             logger.debug("Parsing DBT Data Models")
             manifest_entities = {
-                **dbt_objects.dbt_manifest.nodes,
                 **dbt_objects.dbt_manifest.sources,
```

Comment:
We'll be giving more preference to the `model` nodes than the `source` nodes. For this we'll process the `source` nodes first and then the `model` nodes so that the later metadata will persist.